src/styles/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition: transform 0.3s ease;
+  color: $black;
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
Scaling the card to 120% on hover might cause layout issues if the surrounding elements do not have enough space. Consider using a smaller scale or ensuring that the layout can accommodate the increased size.